src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.cc

```diff
@@ -96,21 +96,19 @@ static void log_address_sorting_list(const ServerAddressList& addresses,
   for (size_t i = 0; i < addresses.size(); i++) {
     char* addr_str;
     if (grpc_sockaddr_to_string(&addr_str, &addresses[i].address(), true)) {
-      gpr_log(GPR_DEBUG, "c-ares address sorting: %s[%" PRIuPTR "]=%s",
-              input_output_str, i, addr_str);
+      GRPC_CARES_TRACE_LOG("c-ares address sorting: %s[%" PRIuPTR "]=%s",
+                           input_output_str, i, addr_str);
       gpr_free(addr_str);
     } else {
-      gpr_log(GPR_DEBUG,
-              "c-ares address sorting: %s[%" PRIuPTR "]=<unprintable>",
-              input_output_str, i);
+      GRPC_CARES_TRACE_LOG("c-ares address sorting: %s[%" PRIuPTR
+                           "]=<unprintable>",
+                           input_output_str, i);
     }
   }
 }
 
 void grpc_cares_wrapper_address_sorting_sort(ServerAddressList* addresses) {
-  if (grpc_trace_cares_address_sorting.enabled()) {
```

Comment:
We should keep this check and leave the log messages in `log_address_sorting_list()` using `gpr_log(GPR_INFO)` instead of `GRPC_CARES_TRACE_LOG()`.  That way, we're not doing extra work we don't need when the tracer is disabled.

Reply:
done